lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+---
+name: Parameters
+field_name_base_url: https://hl7.org/fhir/r4/parameters-definitions.html#Parameters
+fields:
+
+  - name: resourceType
+    required: 'Yes'
+    type: string
+    description: The type of the FHIR resource.
+    action: modify
+    example: |
+      {
+        "resourceType": "Parameters"
+      }
+    note: resourceType must be Parameters
+    url: https://hl7.org/fhir/r4/json.html#resources
+
+  - name: parameter
+    required: 'Yes'
+    type: List of BackboneElement
+    description: Parameters defined by the operation.
+    action: modify
+    url: https://hl7.org/fhir/r4/parameters-definitions.html#Parameters.parameter
+
+  - name: status
+    required: 'Yes'
+    type: code
+    description: The current state of the charge.
+    action: modify
+    example: |
+      {
+        "status": "unknown"
+      }
+    note: >
+      <ul>
+        <li>The status must be set to `unknown`</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.status
+
+  - name: subject
+    required: 'Yes'
+    type: Reference
+    description: The patient associated with the charge.
+    action: modify
+    example: |
+      {
+        "subject": {
+          "reference": "Patient/12768550"
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be a Patient reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.subject
+
+  - name: context
+    required: 'Yes'
+    type: Reference
+    description: The encounter associated with the charge.
+    action: modify
+    example: |
+      {
+        "context": {
+          "reference": "Encounter/97965088"
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be an Encounter reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.context
+
+  - name: occurrenceDateTime
+    required: 'No'
+    type: dateTime
+    description: Date/time when the charged service was applied.
+    action: modify
+    example: |
+      {
+        "occurrenceDateTime": "2021-01-08T10:38:00Z"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.occurrenceDateTime
```

Comment:
This url doesn't resolve, it should be: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.occurrence_x_

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/4595890bdc6f918a94d93cd54ca164dab864e8e0#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R83